peachjam/views/generic_views.py

```diff
@@ -142,7 +142,7 @@ def add_facets(self, context):
         natures = list(
             doc_n
             for doc_n in self.form.filter_queryset(
-                self.get_base_queryset(), exclude="natures"
+                self.get_queryset(), exclude="natures"
```

Comment:
Using get_base_queryset() is the correct thing to do; we don't want post-aggregate filters applied to aggregations.